timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -139,13 +139,14 @@ types:
           requestId:
             type: uuid
             safety: safe
-          lockDescriptors: set<ConjureLockDescriptor>
+          lockDescriptors: list<ConjureLockDescriptor>
           acquireTimeoutMs:
             type: integer
             safety: safe
           clientDescription:
             type: optional<string>
             safety: unsafe
+          metadata: optional<ConjureLockRequestMetadata>
```

Comment:
Interestingly enough, even though https://github.com/palantir/conjure/blob/master/docs/spec/wire.md#42-servers-reject-unknown-fields dictates that servers must reject unknown fields, all conjure object builder are actually annotated with `@JsonIgnoreProperties(ignoreUnknown = true)` meaning that they ignore all unknown properties. If we trust the spec, this change would require a product dependency, but the generated conjure code says otherwise.